cli/pkg/digger/digger.go

```
@@ -364,27 +364,27 @@ func run(command string, job orchestrator.Job, policyChecker policy.Checker, org
 
 		// this might go into some sort of "appliability" plugin later
 		isMergeable, err := prService.IsMergeable(*job.PullRequestNumber)
 		if err != nil {
 			msg := fmt.Sprintf("Failed to check if PR is mergeable. %v", err)
-			return nil, msg, fmt.Errorf(msg)
+			return nil, msg, fmt.Errorf("%s", msg)
 		}
 		slog.Info("PR status Information", "mergeable", isMergeable, "merged", isMerged, "skipMergeCheck", job.SkipMergeCheck)
 		if !isMergeable && !isMerged && !job.SkipMergeCheck {
 			comment := reportApplyMergeabilityError(reporter)
 			prService.SetStatus(*job.PullRequestNumber, "failure", job.ProjectName+"/apply")
```

golangci-lint (1.64.8)

Error return value of `prService.SetStatus` is not checked

(errcheck)